website/docs/r/virtual_hub_connection.html.markdown

```diff
@@ -58,9 +58,9 @@ The following arguments are supported:
 
 ---
 
-* `hub_to_vitual_network_traffic_allowed` - (Optional) Is the Virtual Hub traffic allowed to transit via the Remote Virtual Network? Changing this forces a new resource to be created.
+* `hub_to_vitual_network_traffic_allowed` - (Optional / **Deprecated** ) Is the Virtual Hub traffic allowed to transit via the Remote Virtual Network? Changing this forces a new resource to be created.
 
-* `vitual_network_to_hub_gateways_traffic_allowed` - (Optional) Is Remote Virtual Network traffic allowed to transit the Hub's Virtual Network Gateway's? Changing this forces a new resource to be created.
+* `vitual_network_to_hub_gateways_traffic_allowed` - (Optional / **Deprecated** ) Is Remote Virtual Network traffic allowed to transit the Hub's Virtual Network Gateway's? Changing this forces a new resource to be created.
```

Comment:
arguably since this is non-functional we could remove this:

```suggestion
```

Reply:
How about just keep this (as user is still allowed to assign it) but change the document so something like:

`Due to a breaking behavioural change in the Azure API this property is no longer functional and will be removed in version 3.0 of the provider`